newrelic_lambda_cli/constants.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
```

Comment:
The `enum` module was added in Python 3.4 and this CLI targets >3.3. We may drop support for Python <3.5 at some point. But until then we can't use this module.

Reply:
Ahhh, gotchya, thanks for explaining that :)